OpenID_Connect/get_oidc_config.sh

```diff
@@ -1,6 +1,23 @@
 #!/bin/bash
 DEFAULT_SDC_API_URL='https://app.sysdigcloud.com'
 
+SYSDIG_PRODUCT=""
+PRODUCT_URL=""
+while [ "z${SYSDIG_PRODUCT}" = "z" ] ; do
```

Comment:
please use consistent test blocks. For instance this could be `[[ -z ${SYSDIG_PRODUCT} ]]`